crates/matrix-sdk-ui/src/timeline/event_item/content/reply.rs

```diff
@@ -135,11 +135,13 @@ impl RepliedToEvent {
         let content = match event.original_content() {
             Some(content) => match content {
                 AnyMessageLikeEventContent::RoomMessage(c) => {
-                    // Assume we're not interested in reactions and thread info in this context:
+                    // Assume we're not interested in aggregations in this context:
                     // this is information for an embedded (replied-to) event, that will usually not
                     // include detailed information like reactions.
```

Comment:
Now that the function has four `None` parameters, maybe it's a sign that the `content` bound above should be a `MsgLikeKind`, not a `MsgLike`? (can be a future refactor)